src/modules/ROOT/pages/changelog/202303/05.json

```diff
@@ -0,0 +1,162 @@
+{
+  "meta": {
+    "releasedOn": "2023.08.24"
+  },
+  "added": [
+    {
+      "what": [
+        "the option to configure whether module field value is on the same row as its label in the page block."
+      ],
+      "why": [
+        "to provide the user with the option to set the position of a field value."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/pull/1339"
+      ]
+    },
+    {
+      "what": [
+        "case insensitive search in a record list if MySQL database is used."
+      ],
+      "why": [
+        "to make searching of records easier by returning similar results to the searching criteria."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/pull/1355"
+      ]
+    }
+  ],
+  
+  "changed": [
+    {
+      "what": [
+        "page navigation to stop loading previous resources in case we've navigated to a new page."
+      ],
+      "why": [
+        "this change reduces the amount of needless processing since the old resources would be discarded right away."
```

Comment:
I think this will sound off when rendered; could be wrong tho

Reply:
it does. I changed it